src/main/java/twilightforest/world/GTGenCaveStalactite.java

```diff
@@ -0,0 +1,28 @@
+package twilightforest.world;
+
+import net.minecraft.block.Block;
+import net.minecraft.world.World;
+
+import gregtech.common.blocks.GT_TileEntity_Ores;
+
+public class GTGenCaveStalactite extends TFGenCaveStalactite {
+
+    boolean isSmallOre;
+    boolean air;
+
+    /**
+     * Initializes a stalactite builder
+     */
+    public GTGenCaveStalactite(Block blockType, int meta, float size, int maxLength, int minHeight, boolean isSmallOre,
+            boolean air) {
+        super(blockType, meta, size, maxLength, minHeight);
+        this.isSmallOre = isSmallOre;
+        this.air = air;
+    }
```

Comment:
Note that unless you want to allow variants the block will always be the same, I think GregTech_API.sBlockOres1.